ipaserver/install/server/upgrade.py

```diff
@@ -2031,6 +2045,10 @@ def upgrade_configuration():
     # has been altered
     named_conf_changes = (
                           named_remove_deprecated_options(),
+                          bindinstance.named_add_ext_conf_file(
+                              paths.NAMED_CUSTOM_CFG_SRC,
+                              paths.NAMED_CUSTOM_CONFIG),
+                          named_add_ipa_ext_conf_include(),
```

Comment:
This unprotected call will cause upgrade failure on non-DNS installations.
For instance:
1) install FreeIPA rpm packages, all, but `freeipa-server-dns`;
2) install IPA master without DNS;
2) run ipa-server-upgrade;
3) procedure will fail because `named_add_ext_conf_file` request `NAMED_USER` from passwd db, which in turn is added on `%pre` script of bind-pkcs11 (https://src.fedoraproject.org/rpms/bind/blob/master/f/bind.spec#_1048)

Please, add a wrapper for `named_add_ext_conf_file` into upgrade script like as you did it in `named_add_ipa_ext_conf_include`.